### PR TITLE
New UFC weight classes cheat sheet

### PR DESCRIPTION
**Pull request by edgesince84**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Provide an overview of the changes this pull request introduces.

###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/ufc_weight_classes_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/ufc-weight-classes.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+		"sourceName": "UFC",
+		"sourceUrl": "http://www.ufc.com/discover/sport/weight-classes"
+	},
+
+	"aliases": [
+		"mma weights",
+		"ufc weight classes",
+		"ufc weight divisions",
+		"ufc weight",
+		"ufc weight help"
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/ufc-weight-classes.json]: Please remove this line - the word `help` is included as a default cheat sheet trigger word.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=ufc weight classes cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/ufc_weight_classes_cheat_sheet_mobile.png?EIyZ3c4TG4)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=ufc weight classes cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/ufc_weight_classes_cheat_sheet_mobile.png?GpAroBPQpo)

**Pull request merged**